### PR TITLE
e2e test for search on values

### PR DESCRIPTION
**Pull request by malathi13**

### CODE REVIEW

File: ui/tests/e2e/test.js [hunk 1]
```diff
@@ -168,6 +168,27 @@ describe("End-to-end", () => {
     await waitForFacetsUpdate(3500);
   });
 
+  test("Search box - values", async () => {
```
Comment (melissachang) [inline ui/tests/e2e/test.js]: Search box - select row with facet value

Comment (melissachang) [inline ui/tests/e2e/test.js]: Delete `test("Search box - click on drop-down row")`. This test already clicks on drop-down row.

File: ui/tests/e2e/test.js [hunk 2]
```diff
@@ -168,6 +168,27 @@ describe("End-to-end", () => {
...
+    // Select first result
+    await page.waitForXPath("//span[contains(text(), 'Add')]");
+    let results = await page.$x("//span[contains(text(), 'Add')]");
+    await results[0].click();
+
+    // Assert Relationship facet is added
+    await waitForFacetCard("Relationship");
+    await assertFacet("Relationship", "1", "mother", "831");
+
+    // Make sure non-selected facet values are gray.
```
Comment (melissachang) [inline ui/tests/e2e/test.js]: opt - These tests should really be asserting that a facet value is checked, not that a different value is gray. I could have sworn that was somewhere in this test, but I can't find it now. Up to you -- add assertFacetValueSelect() and have this test and others use it.

Comment (malathi13) [reply ui/tests/e2e/test.js]: I am not sure which other tests need this, I changed this test to check if the value is selected.

**Pull request merged**